<ui version="4.0" >
 <class>AddLegacyContactDialogClass</class>
 <widget class="QDialog" name="AddLegacyContactDialogClass" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>84</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <item>
    <widget class="QLabel" name="lblDescription" >
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lblPrompt" >
       <property name="text" >
        <string>Contact ID:</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lnePrompt" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dbbButtons" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
